docker/package/systemd/tezos-node.service

```diff
@@ -0,0 +1,15 @@
+# SPDX-FileCopyrightText: 2020 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+[Unit]
+After=network.target
+Description=Tezos node
+
+[Service]
+EnvironmentFile=/etc/defaults/tezos-node.conf
+Environment="TEZOS_CLIENT_UNSAFE_DISABLE_DISCLAIMER=Y"
```

Comment:
Oh, btw, I think we shouldn't unconditionally disable this disclaimer in our packages. For us this disclaimer is annoying and not interesting, but for random users it can be helpful. There is a reason why it exists and I think these packages are not the right place to say that this disclaimer shouldn't exist.

Reply:
hmm, actually, I don't think that this disclaimer is used in these binaries, so it indeed makes sense to remove it

Reply:
Also, I've noticed that default verbosity is `notice`, not `info` :pled:

Reply:
If default is `notice`, does setting `"TEZOS_LOG=* -> notice"` do anything? If we don't set `TEZOS_LOG`, will it be `notice` anyway? I think it's better to just use default verbosity.